### ISSUE TITLE
Reset password function not working yet - need to implement

### ISSUE THREAD

**Issue by doricon** [paleobot/pbot-dev#316]
Need to make the reset password function work. There is a button for it on the login page, but nothing happens. Not sure if it is a bug or just not yet implemented.

**Comment by aazaff** [comment]
Yes, this is not implemented yet. We have discussed this many times in the meeting as last Core item we will actually address. This requires an SMTP server to be set up.
